Sources/WWDCNotes/WWDCNotes.docc/WWDC24/WWDC24-10166-Build-compelling-spatial-photo-and-video-experiences.md

```diff
@@ -8,11 +8,113 @@ Learn how to adopt spatial photos and videos in your apps. Explore the different
    @CallToAction(url: "https://developer.apple.com/wwdc24/10166", purpose: link, label: "Watch Video (21 min)")
 
    @Contributors {
-      @GitHubUser(<replace this with your GitHub handle>)
+      @GitHubUser(halmueller)
    }
 }
 
-😱 "No Overview Available!"
+## Chapters
```

Comment:
Just FYI: I'm planning to automate the creation of these timestamps in the future. The info is all there in the metadata, I just haven't implemented the code for this yet.  But for now, this is just fine, I'll remove it when my feature is implemented.